src/python/tests/core/platforms/android/sanitizer_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for sanitizer functions."""
+
+import os
+
+from platforms.android import adb
+from platforms.android import constants
+from platforms.android import sanitizer
+from platforms.android import settings
+from system import environment
+from tests.test_libs import android_helpers
+from tests.test_libs import helpers as test_helpers
+
+DATA_PATH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), 'sanitizer_data')
+
+
+class GetOptionsFilePathTest(android_helpers.AndroidTest):
+  """Tests get_options_file_path."""
+
+  def test_regular_build(self):
```

Comment:
nit: add docstrings for these test methods?

Reply:
done